src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java

```diff
@@ -10,20 +10,30 @@
 import org.apache.commons.lang.StringUtils;
 import org.openqa.selenium.WebDriverException;
 
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
 public class UrlBuilder extends BaseUrlBuilder {
 
   public static final String WIKI_NAME_FANDOM_SUFFIX = "fandom";
+  private static final Set<String>
+      FANDOM_EXCULUDED_WIKIS =
+      new HashSet<>(Arrays.asList(new String[]{"community"}));
   private final String wikiName;
   private Boolean forceHttps;
   private Boolean forceLanguageInPath;
   private String language;
 
   private UrlBuilder(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "UrlBuilder" constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)